themes/wilson_theme_starterkit/templates/paragraph/paragraph--stat.html.twig

```diff
@@ -0,0 +1,55 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display an accordion item paragraph.
+ *
+ * Available variables:
+ * - paragraph: Full paragraph entity.
+ *   Only method names starting with "get", "has", or "is" and a few common
+ *   methods such as "id", "label", and "bundle" are available. For example:
+ *   - paragraph.getCreatedTime() will return the paragraph creation timestamp.
+ *   - paragraph.id(): The paragraph ID.
+ *   - paragraph.bundle(): The type of the paragraph, for example, "image" or "text".
+ *   - paragraph.getOwnerId(): The user ID of the paragraph author.
+ *   See Drupal\paragraphs\Entity\Paragraph for a full list of public properties
+ *   and methods for the paragraph object.
+ * - content: All paragraph items. Use {{ content }} to print them all,
+ *   or print a subset such as {{ content.field_example }}. Use
+ *   {{ content|without('field_example') }} to temporarily suppress the printing
+ *   of a given child element.
+ * - attributes: HTML attributes for the containing element.
+ *   The attributes.class element may contain one or more of the following
+ *   classes:
+ *   - paragraphs: The current template type (also known as a "theming hook").
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_paragraph()
+ *
+ * @ingroup themeable
+ */
+#}
+{% block paragraph %}
+  <h3 class="font-bold tracking-wider">
+    {% apply spaceless %}
+      {% if content.field_stat_prefix|render %}
+        <span class="prefix">{{ content.field_stat_prefix|field_value }}</span>
+      {% endif %}
+      <span class="value{% if paragraph.field_stat_animate.value %} countup{% endif %}">{{ content.field_stat_value|field_value }}</span>
```

Comment:
Out of curiosity - when using a screen reader, will that read zero/the counted up numbers/only the final value? 🤔 

Perhaps we could make the animated value invisible to screen readers, and print out a static value that's invisible to sighted users?

Reply:
Also a good point. I'm not sure what it would read. I suppose it depends on if the browser considers the element to have scrolled in to view, otherwise it will read zero. I like the idea of having two elements,